src/main/java/org/fundacionjala/coding/wilson/Dna.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.coding.wilson;
+
+/**
+ *
+ */
+public final class Dna {
+    /**
+     *
+     */
+    private Dna() {
+
+    }
+
+    /**
+     *
+     * @param dna Given type String
+     * @return chars String
+     */
+    public static String makeComplement(final String dna) {
+        char[] chars = dna.toCharArray();
+
+        for (int i = 0; i < chars.length; i++) {
+            chars[i] = getComplement(chars[i]);
+        }
+        return new String(chars);
+    }
+
+    /**
+     *
+     * @param c is char
+     * @return change
+     */
+    private static char getComplement(char c) {
+        switch (c) {
```

Comment:
Don't use a switch case
Consider using a Map or take advantage of String methods ( replaceAll )